autoload/SpaceVim/mapping.vim

```diff
@@ -145,7 +145,7 @@ function! SpaceVim#mapping#close_current_buffer() abort
   let bn = bufnr('%')
   let f = ''
   if getbufvar(bn, '&modified', 0)
-    redraw!
+    redraw
```

Comment:
why remove `!`? with `!` we can clear the screen first.

Reply:
With `!`, I saw a slight flash after pressing `y` on a slow laptop. I am wondering is it necessary to put a `!` after the `redraw`, as `redraw` could also do the job.

Reply:
without `!`, it will not clear screen. sometimes you will see unwanted chars in the cmdline.